### PR TITLE
Adding simple SDX integration

### PR DESCRIPTION
**Pull request by payneBrandon**
Adding SDX deposit

### CODE REVIEW

File: compose.yml [hunk 1]
```diff
@@ -85,6 +85,26 @@ services:
...
+      SDW_DESTINATION_URL: ${SDW_DESTINATION_URL}
+      SDW_API_KEY: ${SDW_API_KEY}
+      SDW_EMAIL_LIST: ${SDW_EMAIL_LIST}
+      SDW_EMAIL_FROM: ${SDW_EMAIL_FROM}
+      SPRING_MAIL_HOST: ${SPRING_MAIL_HOST}
+      SPRING_MAIL_PORT: ${SPRING_MAIL_PORT}
+    depends_on:
+      - ode
+    volumes:
+      - ${DOCKER_SHARED_VOLUME}:/asn1_codec_share
```
Comment (dmccoystephenson) [inline compose.yml]: This volume mount probably isn't necessary.

**Review by dmccoystephenson — COMMENTED** [review]
Should we update the `start_ode.sh` script to check that the sdw-depositor container starts up like it does with the other containers?

File: sample.env
Comment (dmccoystephenson) [inline sample.env:1]: The `generate_tim.sh` script seemed to have trouble with the comments, so I had to remove them to get the script to import the .env file correctly.

**Review by dmccoystephenson — DISMISSED** [review]
This looks good & I was able to successfully deposit to the SDX with this setup (after removing comments from the .env).

I did notice that none of the sample TIMs are set up to go to the SDX. Do we want to include a sample TIM file that is destined for the SDX?

File: sample.env
Comment (payneBrandon) [reply sample.env:1]: Updated README to reflect format

File: compose.yml (hunk 1 above)
Comment (payneBrandon) [reply compose.yml]: Removed

**Comment by payneBrandon** [comment]
> Should we update the `start_ode.sh` script to check that the sdw-depositor container starts up like it does with the other containers?

Done

**Review by dmccoystephenson — CHANGES_REQUESTED** [review]
The SDX deposit still works as expected, but I am seeing the following when I use the new sample TIM:

>TIM not found in ODE logs. Please try again.

I noticed that the `generate_tim.sh` script looks for 'Encoded message - phase 1:', but only  'Encoded message - phase 2:' is getting printed to the console when submitting the sample SDX TIM. The script should probably be updated to account for both cases.

**Comment by payneBrandon** [comment]
> The SDX deposit still works as expected, but I am seeing the following when I use the new sample TIM:
> 
> > TIM not found in ODE logs. Please try again.
> 
> I noticed that the `generate_tim.sh` script looks for 'Encoded message - phase 1:', but only 'Encoded message - phase 2:' is getting printed to the console when submitting the sample SDX TIM. The script should probably be updated to account for both cases.

I'm not sure how useful it would be to grab the encoded value from there since it's already submitted to the SDX and in a doubly-encoded format to boot. But I've remove the phase number specificity to get it to save anyway

**Review by dmccoystephenson — APPROVED** [review]
Looks good to me!

**Pull request merged**